components/image/image.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image | Sapota UI</title>
+    <link rel="stylesheet" href="../../docs/css/index.css" />
+    <link rel="stylesheet" href="./image.css" />
+</head>
+<body>
+    <div class="wrapper-image">
+        <h2>Image</h2>
+
+        <h3>Responsive Image</h3>
+        <div class="img-container">
+            <img class="img-responsive" src="https://picsum.photos/500" alt="">
```

Comment:
Suggestion: images should be nested inside a figure tag as it is a best practice. [Check Here](https://www.learningjournal.guru/article/html5/html5-when-to-use-figure-tag/)
Everything else looks good

Reply:
Thanks for suggestion, will keep in mind for later